Sources/KituraContracts/Contracts.swift

```diff
@@ -22,7 +22,7 @@
  and server side (eg Kitura) of the request (typically a HTTP REST request).
 
  ### Usage Example: ###
- ````
+ ````swift
```

Comment:
Why are the parameters not defined on the init() methods which pass in parameters? 
i.e. `public init(rawValue: Int)` and `public init(rawValue: Int, reason: String)`.

It also looks like a new initializer has been added since this was generated `public init<Body: Codable>(_ base: RequestError, body: Body)` whose parameters also need documenting.